### ISSUE TITLE
Accuracy Goes Down And Loss Increases After 19th Epoch

### ISSUE THREAD

**Issue by xuzhang5788** [tsudalab/ChemTS#8]
After 19th epoch, I got
loss: 0.3925 - acc: 0.8709 - val_loss: 0.3650 - val_acc: 0.8786

but after 20th epoch, loss increases a lot
loss: 0.8816 - acc: 0.7350 - val_loss: 0.7405 - val_acc: 0.7701

then at 21th epoch
loss: 0.7743 - acc: 0.7613 - val_loss: 0.6743 - val_acc: 0.7912

Is it normal? Did you change the learning rate after 20 epochs? What is the final loss and accuracy after 100th epoch when you trained your model?

After 100 epochs training, I got 
loss: 0.5779 - acc: 0.8223 - val_loss: 0.5234 - val_acc: 0.8355

Is it acceptable or I need to train more epochs?

Thanks

**Comment by yangxiufengsia** [comment]
Hi @xuzhang5788, If you want to obtain fast convergence, please reduce the hidden-dimensions of GRU(e.g. reduce 512 to 256).
